images/runtime/dotnetcore/3.0/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Startup script generator
+FROM golang:1.11-stretch as startupCmdGen
+# GOPATH is set to "/go" in the base image
+WORKDIR /go/src
+COPY src/startupscriptgenerator/src .
+ARG GIT_COMMIT=unspecified
+ARG BUILD_NUMBER=unspecified
+ENV GIT_COMMIT=${GIT_COMMIT}
+ENV BUILD_NUMBER=${BUILD_NUMBER}
+RUN ./build.sh dotnetcore /opt/startupcmdgen/startupcmdgen
+
+FROM mcr.microsoft.com/dotnet/core/sdk:3.0 AS tools-install
```

Comment:
Since currently these tools are only available as part of SDK, we install the tools in a directory and later copy them to the final runtime image

Reply:
nit:can you add this comment in the dockerfile also?